src/rpc/mining.cpp

```diff
@@ -419,8 +419,8 @@ static RPCHelpMan getmininginfo()
                     RPCResult::Type::OBJ, "", "",
                     {
                         {RPCResult::Type::NUM, "blocks", "The current block"},
-                        {RPCResult::Type::NUM, "currentblockweight", /*optional=*/true, "The block weight of the last assembled block (only present if a block was ever assembled)"},
-                        {RPCResult::Type::NUM, "currentblocktx", /*optional=*/true, "The number of block transactions of the last assembled block (only present if a block was ever assembled)"},
+                        {RPCResult::Type::NUM, "currentblockweight", /*optional=*/true, "The block weight (include coinbase reserved weight) of the last assembled block (only present if a block was ever assembled)"},
```

Comment:
9990552514328da45c19ed294c5bc79562aa12bc nit: "including"

Reply:
fixed, even better I clarified that it also included the block header and txs count.